apis/nucleus/src/components/Cell.jsx

```diff
@@ -594,6 +604,7 @@ const Cell = forwardRef(
             xs
             style={{
               height: '100%',
+              ...(bodyPadding ? { padding: bodyPadding } : {}),
```

Comment:
We could add a defined className here as well, making sure it can be customized.
ex:
```
/**
 * @interface
 * @extends HTMLElement
 */
const CellBody = {
  /** @type {'njs-cell-body'} */
  className: 'njs-cell-body',
};
```

Reply:
done